pkg/kubelet/kubelet.go

```diff
@@ -1561,7 +1561,7 @@ func (kl *Kubelet) StartGarbageCollection() {
 			if prevImageGCFailed {
 				klog.ErrorS(err, "Image garbage collection failed multiple times in a row")
 				// Only create an event for repeated failures
-				kl.recorder.Eventf(kl.nodeRef, v1.EventTypeWarning, events.ImageGCFailed, err.Error())
+				kl.recorder.Eventf(kl.nodeRef, v1.EventTypeWarning, events.ImageGCFailed, "%s", err.Error())
```

Comment:
Why was this change added? Was the error not reported?

Reply:
let's use `kl.recorder.Event` instead then

Reply:
Done, thanks for the ping and apologies for missing the comment the first time.